content/blog/2022/12/camel4roadmap/index.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "Roadmap to Camel v4"
+date: 2022-12-16
+draft: false
+authors: [davsclaus]
+categories: ["Roadmap"]
+preview: "Roadmap to Apache Camel v4 and impact on Camel v3"
+---
+
+Apache Camel v4 is on the way for 1st half in 2023.
+
+The need for Camel v4 is mainly driven by Java open source projects migrating from `javax` to `jakarta` APIs
+and to keep up with popular runtimes such as Spring Boot and Quarkus.
+
+## Primary Goals 
+1. Migrate from `javax` -> `jakarta` (JEE 10)
+2. Java 17 as minimum
+3. Spring Framework 6
+4. Spring Boot 3
+5. Quarkus 3
+
+## Release Goals
+6. Release only what is ready (JEE10 / Java17)
+   This means that Camel components that are not ready (yet) will be dropped in a release until they are ready.
+7. Release Camel Core + Camel Spring Boot together
+8. Move Camel Karaf to Apache Karaf as karaf-camel sub-project 
+
+## Major Goals
+9. Support Java 17 features such as records, multiline strings, and what else
```

Comment:
Instead of "and what else", I'd suggest "and others", or something specific, maybe pattern matching or switch expressions.